pkg/metrics/metrics.go

```diff
@@ -66,15 +71,15 @@ var pagingAlerts = []string{
 type Metrics interface {
 	UpdateMetricValidationFailed(string)
 	UpdateMetricValidationSucceeded(string)
-	UpdateMetricClusterCheckFailed(string)
-	UpdateMetricClusterCheckSucceeded(string)
+	UpdateMetricUpgradeoperatorHealthcheckSucceeded(string)
```

Comment:
```suggestion
	UpdateMetricHealthcheckSucceeded(string)
```

Reply:
Following the convention of other methods in this interface, I think mentioning "Upgradeoperator" isn't necessary and makes the name too long. Could you refactor the mocks and it's references accordingly?